cpp/ps0003/cpp_ps0003_solution.cpp

```diff
@@ -2,11 +2,17 @@
 using namespace std;
 int main()
 {
+	// prompting user for number 2/2
 	int x;
-	cout << "enter a namber\n";
+	cout << "enter a number\n";
 	cin >> x;
+
+	// checking if the number is odd or even 3/3
 	if (x % 2 == 0)
 		cout << "the number is even\n";
 	else
 		cout << "the number is odd\n";
-}
\ No newline at end of file
+	

```

Comment:
- [x] used modulo operation by 2 `%2` to check if the number is even or odd
- [x] number is even when modulo result returns `0` and odd when returns `1`
- [x] used `std::cout` in a right way to print result in console